server/src/router.ts

```diff
@@ -315,3 +339,25 @@ const spawnRates: { [timeSeconds: number]: number } = {
   80: 0.6,
   100: 0.7,
 };
+
+const removeImageAfter1Min = () => {
+  if (backgroundState.imageInterval) {
+    clearInterval(backgroundState.imageInterval);
+  }
+
+  backgroundState.imageInterval = setTimeout(() => { 
```

Comment:
imageInterval would be clearer if named imageTimeout since it's technically a timeout not an interval